app/views/docs/changes.json.jbuilder

```diff
@@ -1,7 +1,7 @@
 json.changes @changes do |change|
   json.druid change.druid
   json.latest_change change.published_at.iso8601
-  json.true_targets change.release_tags.where(release_type: true).map(&:name) if change.release_tags.where(release_type: true).present?
+  json.true_targets change.release_tags.where(release_type: true).map(&:name) | Settings.ALWAYS_SEND_TRUE_TARGET
   json.false_targets change.release_tags.where(release_type: false).map(&:name) if change.release_tags.where(release_type: false).present?
```

Comment:
i don't think this works as intended if `Settings.ALWAYS_SEND_TRUE_TARGET` is `[]` or `nil`. can we enforce that there's at least one value in this constant array somehow? maybe in initializers/config.rb?